lib/tasks/audit_code.rake

```diff
@@ -403,6 +373,17 @@ def get_release
   release
 end
 
+def get_reviewer(prompt = false)
+  reviewer = ENV['reviewed_by']
+  if reviewer.nil?
+    require 'rugged'
+    repo = Rugged::Repository.new('.')
```

Comment:
What's the behaviour here is the user is in an `svn` working copy? We don't want it to crash...

Reply:
I will check against a svn repository to get an answer.